kai-service/mock-client.py

```diff
@@ -176,35 +271,110 @@ async def main():
           }
       }
   }""",
-            "line_number": 36,
-            "analysis_message": """Enterprise Java Beans (EJBs) are not supported in Quarkus. CDI must be used.
-Please replace the `@MessageDriven` annotation with a CDI scope annotation like `@ApplicationScoped`.""",
+            "application_name": "test_app",
+            "incidents": [
+                {
+                    "violation_name": "jms-to-reactive-quarkus-00010",
+                    "ruleset_name": "kai/quarkus",
+                    "incident_snip": """  1  /*
```

Comment:
Just a FYI, I think we should drop incident_snip.

I don't believe we want to send incident_snip in to our API or consider it with the prompt builder.

Reply:
could we fix this in a followup for all the endpoints?

Reply:
Oh yeah, dont need to address now, more just information. 
@JonahSussman and I spoke about codeSnip yesterday and there was some confusion if we should use it.

My thinking is we should ignore codeSnip for Kai purposes and assume the input file contents we are sent are what we want to focus on.

All good to merge this now as-is and we can adjust for codeSnip later.